toml.md

```diff
@@ -38,23 +38,21 @@ should be easy to parse into data structures in a wide variety of languages.
 
 - TOML is case-sensitive.
 - A TOML file must be a valid UTF-8 encoded Unicode document.
-- Whitespace means tab (0x09) or space (0x20).
-- Newline means LF (0x0A) or CRLF (0x0D 0x0A).
+- Whitespace means tab (U+0009) or space (U+0020).
+- Newline means LF (U+000A) or CRLF (U+000D U+0x0A).
```

Comment:
```suggestion
- Newline means LF (U+000A) or CRLF (U+000D U+000A).
```
A very minor error here! :)

(IMO best would be having these unrelated cosmetic changes in a separate PR)

Reply:
Since it's germane to what we're discussing here, let's roll it into this PR.